### DISCUSSION TITLE
Move outline into the text editor

### DISCUSSION THREAD

**Discussion by royfrancis** [posit-dev/positron#4357] [Feedback]
This is just an opinion. The outline in VS code is something that has bothered me for a long time. It is in a separate panel for whatever reason. It's not ideal having to switch from the text editor to another panel to jump around in the document. I think it should ideally be where the minimap is. Or adjacent to it. Personally, I think the minimap is useless, so I wouldn't mind it being replaced with outline.

![Screenshot 2024-08-14 at 21 41 31](https://github.com/user-attachments/assets/72672312-8e9c-4b39-b56d-730965586028)

This is one of the things that RStudio got right.

![Screenshot 2024-08-14 at 21 46 18](https://github.com/user-attachments/assets/177c6537-547c-4b3c-814b-8f7c5306204c)

Since this is an opportunity to fix these little bugging issues, I just wanted to put it out there.

**Comment by seeM** [comment]
You can do this by setting `editor.minimap.enabled` to false, opening the right side panel, and dragging the Outline pane onto the right side panel.

<img width="1072" alt="image" src="https://github.com/user-attachments/assets/494abea6-aca2-4be8-8719-325b449cf749">
